lib/beaker-hostgenerator/data.rb

```diff
@@ -164,6 +164,18 @@ def osinfo
             'template' => 'amazon-7-arm64'
           }
         },
+        'archlinuxrolling-64' => {
+          :general => {
+            'platform'           => 'archlinux-rolling-x64',
+            'packaging_platform' => 'archlinux-rolling-x64'
```

Comment:
I learned that if `packaging_platform` is the same as `platform`, it can be left out.